src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Ball {
+    private int number;
+    private String color;
+
+    protected Ball(int numberRandom, String colorRandom) {
+        number = numberRandom;
+        color = colorRandom;
+    }
```

Comment:
```suggestion
    public Ball(int number, String color) {
        this.number = number;
        this.color = color;
    }
```

This is common approach for constructor with parameters

Reply:
private final Random random = new Random();
    private ColorSupplier colorSupplier = new ColorSupplier();

I had this. Why it was not correct??????